### PR TITLE
Enforce one-sentence-per-line rule in adoc

### PR DESCRIPTION
**Pull request by elefeint**
Bring existing adoc into compliance with one-sentence-per-line adoc best practice.

### CODE REVIEW

**Comment by elefeint** [comment]
Many attempts at sensible regexes, followed by ctrl+z. I only did the "indent=0" stuff manually because it did not always need to be done.

**Comment by elefeint** [comment]
That's all right -- I was not sure it was a good idea, that's why I did not submit it last year.
I'll remove the rule and workarounds, and leave just the existing fixes.

**Comment by elefeint** [comment]
@artembilan I think I removed all the bandaids. PTAL?

File: spring-cloud-gcp-samples/spring-cloud-gcp-trace-sample/README.adoc [hunk 1]
```diff
@@ -22,8 +22,7 @@ Run the example from Maven:
 $ mvn spring-boot:run
 ----
 
-Browse to `http://localhost:8080`. This will trigger the `ExampleController.work()` method,
-which in turn will call other services, and also a remote RESTful call to `ExampleController.meet()` method.
+Browse to `http://localhost:8080`. This will trigger the `ExampleController.work()` method, which in turn will call other services, and also a remote RESTful call to `ExampleController.meet()` method.
```
Comment (artembilan) [inline spring-cloud-gcp-samples/spring-cloud-gcp-trace-sample/README.adoc]: This is missed looks like. There is a period after `8080`. 😉

File: spring-cloud-gcp-samples/spring-cloud-gcp-logging-sample/README.adoc [hunk 2]
```diff
@@ -29,21 +25,18 @@ $ curl --header "x-cloud-trace-context: [Some-Trace-ID]" localhost:8080/log
...
 Select the `Global` resource and look under the `spring.log` channel to see the logged text.
 
-If your application is deployed to Google App Engine (GAE) or Google Kubernetes Engine (GKE),
-you will instead find the logs under the deployment's resource in the Stackdriver UI under the `spring.log` channel.
-You can further narrow down the log by request by using the following filtering text
-that takes advantage of the trace ID label in each logged line. For example, if deployed to App Engine
+If your application is deployed to Google App Engine (GAE) or Google Kubernetes Engine (GKE), you will instead find the logs under the deployment's resource in the Stackdriver UI under the `spring.log` channel.
+You can further narrow down the log by request by using the following filtering text that takes advantage of the trace ID label in each logged line.
+For example, if deployed to App Engine
 (by using `mvn appengine:deploy` in this sample) you can use the following to filter by the trace ID:
```
Comment (artembilan) [inline spring-cloud-gcp-samples/spring-cloud-gcp-logging-sample/README.adoc]: This line has to be connected with the previous one.

File: spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-sample/README.adoc [hunk 3]
```diff
@@ -34,10 +30,7 @@ These POJOs are then retrieved and their contents are printed in the browser.
 Note that the sample application, still running, also provides REST endpoints.
 For example, you can retrieve all `Trade` objects in the repository by using
 `curl http://localhost:8080/trades`, or any specific trade via
-`curl http://localhost:8080/trades/<trader_id>,<trade_id>`. This is achieved using
-https://projects.spring.io/spring-data-rest/:[Spring Data REST] and the `@RepositoryRestResource`
-annotation.
+`curl http://localhost:8080/trades/<trader_id>,<trade_id>`. This is achieved using https://projects.spring.io/spring-data-rest/:[Spring Data REST] and the `@RepositoryRestResource` annotation.
```
Comment (artembilan) [inline spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-sample/README.adoc]: `This is ...` is a new sentence, therefore from new line.

File: spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-sample/README.adoc [hunk 4]
```diff
@@ -1,26 +1,22 @@
...
 +
-Then, uncomment the `spring.cloud.gcp.spanner.credentials.location` property in the
-link:src/main/resources/application.properties[application.properties] file and fill its value with the path to your service account private key on your local file system, prepended with `file:`.
+Then, uncomment the `spring.cloud.gcp.spanner.credentials.location` property in the link:src/main/resources/application.properties[application.properties] file and fill its value with the path to your service account private key on your local file system, prepended with `file:`.
 
-. Run the `SpannerRepositoryExample` or the `SpannerTemplateExample` Spring Boot console apps. This causes the app to generate sample `Trade`
-objects filled with some demo data and persists them to the Cloud Spanner table created earlier.
+. Run the `SpannerRepositoryExample` or the `SpannerTemplateExample` Spring Boot console apps.
+This causes the app to generate sample `Trade` ƒobjects filled with some demo data and persists them to the Cloud Spanner table created earlier.
```
Comment (artembilan) [inline spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-sample/README.adoc]: Typo: there should not be that `f` in cursive 🤷‍♂️

File: spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-sample/README.adoc [hunk 5]
```diff
@@ -1,26 +1,22 @@
...
-Data Cloud Spanner module
-link:../../spring-cloud-gcp-starters/spring-cloud-gcp-starter-data-spanner[Spring Cloud GCP Cloud Spanner Starter].
+Data Cloud Spanner module link:../../spring-cloud-gcp-starters/spring-cloud-gcp-starter-data-spanner[Spring Cloud GCP Cloud Spanner Starter].
 
 = Running the example
 
 
-. Create a new Cloud Spanner instance named "spring-demo". The example application
-will create the database and tables if they do not already exist.
+. Create a new Cloud Spanner instance named "spring-demo". The example application will create the database and tables if they do not already exist.
```
Comment (artembilan) [inline spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-sample/README.adoc]: `The example ...` from new line

File: docs/src/main/asciidoc/vision.adoc [hunk 6]
```diff
@@ -73,7 +73,7 @@ private CloudVisionTemplate cloudVisionTemplate;
 public void processImage() {
   Resource imageResource = this.resourceLoader.getResource("my_image.jpg");
   AnnotateImageResponse response = this.cloudVisionTemplate.analyzeImage(
-      imageResource, Type.LABEL_DETECTION);
+       imageResource, Type.LABEL_DETECTION);
```
Comment (artembilan) [inline docs/src/main/asciidoc/vision.adoc]: I believe this whitespace has to be removed as well.

**Pull request merged**